scripts/rbacgen/main.go

```diff
@@ -0,0 +1,91 @@
+package main
```

Comment:
I really want to make sure we keep this enum synced. We should probably make a more generic way to maintain enum lists.